docs/commands.md

```diff
@@ -0,0 +1,167 @@
+
+[Chinese Ideographs](https://github.com/Nautilus-Cyberneering/chinese-ideographs) is a sample library which follows the [Nautilus Filename Specification](https://github.com/Nautilus-Cyberneering/nautilus-namecodes). It contains some Chinese drawings related to Chinese ideographs.
+
+In the data folder you can see all the images:
+
+```text
+data
+├── 000000
+│   ├── 32
+│   │   ├── 000000-32.600.2.tif
+│   │   └── 000000-32.600.2.tif.dvc
+│   └── 52
+│       ├── 000000-52.600.2.tif
+│       └── 000000-52.600.2.tif.dvc
+...
+├── 000007
+│   ├── 32
+│   │   ├── 000007-32.600.2.tif
+│   │   └── 000007-32.600.2.tif.dvc
+│   └── 52
+└── 000008
+    ├── 32
+    │   ├── 000008-32.600.2.tif
+    │   └── 000008-32.600.2.tif.dvc
+    └── 52
+
+27 directories, 32 files
+```
+
+In fact, it does not contain the image itself but the "pointer" to the file in the remote DVC storage (`.dvc` extensions). [DVC](https://dvc.org/) is a wrapper on top of Git to version and store store binary files. It is an alternative to [Git LFS](https://git-lfs.github.com/). You can get (`pull`) the real images from the remote DVC storage into your local file system and they are ignored in the Git repository.
+
+[DVC](https://dvc.org/) has a command similar to `git diff` called `dvd diff` which give you a list of the changes between two commits.
+
+For example, if you clone that repo:
+
+```shell
+git clone https://github.com/Nautilus-Cyberneering/chinese-ideographs
+cd chinese-ideographs
+```
+
+and you run this command:
+
+```shell
+dvc diff --json 420ea8d 6e9878f
+```
+
+You will obtain this `json` object (it's has been truncated here):
+
+```json
+{                                                                     
+  "added": [],
+  "deleted": [
+    {
+      "path": "data/000000/32/000000-32.600.2.tif"
+    },
+    {
+      "path": "data/000008/32/000008-32.600.2.tif"
+    }
+  ],
+  "modified": [
+    {
+      "path": "data/000001/32/000001-32.600.2.tif"
+    }
+  ],
+  "renamed": []
+}
+```
+
+That means some images were deleted and one image was modified.
+
+The `gold-images-processing` command helps you to handle all changes related to `Gold` images. `Gold` images are identified by their purpose code `32`. We know that the modified image is a `Gold` image because the second code in the name is `32`, following the art work ID: `000001-32.600.2.tif`.
+
+The main goal for this command is to generate `Base` images automatically. When you add a new `Gold` image to a library that image is usually too big, and very often you do not need a very high resolution image. `Base` images are a second type of image that have a lower resolution and can be used for a lot of use cases. The `gold-images-processing` command automatically generates and keep synced the set of `Base` images.
+
+The `gold-images-processing` command also helps to keep the library clean and tidy. These are the list of all tasks.
```

Comment:
"These are the list of all tasks."
You refer to the list with the verb not the tasks therefore:
"This is the list of all tasks."